packages/docs/concepts/connections-and-requests.yaml

```diff
@@ -75,16 +73,28 @@ _ref:
 
             # Requests
 
-            Requests can be defined on any block, and the results of the request are available to any block in the same context. Requests must have an `id`, `type`, `connectionId` field specifying the connection to use, and `properties` defining the request settings. Requests can be called using the [`Request`](/Request) action. Operators in request properties are evaluated every time a request is called.
+            Requests can be defined on any block, and the results of the request are available to any block in the same context. Requests must have an `id`, `type`, `connectionId` field specifying the connection to use, and `properties` defining the request settings. Requests can be called using the [`Request`](/Request) action.
```

Comment:
`and the results of the request are available to any block in the same context` should be removed  and replaced with some like `the results of the request can be access using the _request operator anywhere on the same page`.